instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/TracingClientInterceptor.java

```diff
@@ -56,7 +58,8 @@
       }
     }
 
-    SocketAddress address = result.getAttributes().get(Grpc.TRANSPORT_ATTR_REMOTE_ADDR);
+    URI uri = GrpcUtil.authorityToUri(next.authority());
```

Comment:
Let's just call `URI` constructor itself (and not throw on an invalid one)

https://github.com/grpc/grpc-java/blob/5b6ee1c1766165c85bb58e930a1b8dbbb705b629/core/src/main/java/io/grpc/internal/GrpcUtil.java#L495